RELEASE.md

```diff
@@ -4,6 +4,7 @@
 * `pipeline` now accepts `tags` and a collection of `Node`s and/or `Pipeline`s rather than just a single `Pipeline` object. `pipeline` should be used in preference to `Pipeline` when creating a Kedro pipeline.
 * `pandas.SQLTableDataSet` and `pandas.SQLQueryDataSet` now only open one connection per database, at instantiation time (therefore at catalog creation time), rather than one per load/save operation.
 * Added new command group, `micropkg`, to replace `kedro pipeline pull` and `kedro pipeline package` with `kedro micropkg pull` and `kedro micropkg package` for Kedro 0.18.0. `kedro micropkg package` saves packages to `project/dist` while `kedro pipeline package` saves packages to `project/src/dist`.
+* Added option to `SparkDataSet` to specify a `schema` load argument that allows for supplying a user-defined schema as opposed to relying on the schema inference of Spark.
```

Comment:
These will have to go in a new 0.17.8 release (apologies again...)

Reply:
@lorenabalan I was intending to expand the doc, but I must admit that I could not directly locate where the [example usage](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.spark.SparkDataSet.html) form the website is coming from.